crypto/internal.h

```diff
@@ -159,14 +159,6 @@ extern "C" {
 void OPENSSL_cpuid_setup(void);
 #endif
 
-#if (defined(OPENSSL_ARM) || defined(OPENSSL_AARCH64)) && \
-    !defined(OPENSSL_STATIC_ARMCAP)
-// OPENSSL_get_armcap_pointer_for_test returns a pointer to |OPENSSL_armcap_P|
-// for unit tests. Any modifications to the value must be made after
-// |CRYPTO_library_init| but before any other function call in BoringSSL.
-OPENSSL_EXPORT uint32_t *OPENSSL_get_armcap_pointer_for_test(void);
-#endif
```

Comment:
It's unclear to me why we are removing this. Is there a rational for this/ reason this won't break anything?

Reply:
I'm paraphrasing a bit but we can't have this logic in our FIPS build because the running application could accidentally change the running characteristics of AWS-LC into something that isn't approved. We could keep if for the regular build but we decided to move everything to environment variable for simplicity.

Reply:
Correct, due to FIPS we are not allowed to expose a function that returns a pointer to the capabilities variable.